bigbluebutton-html5/imports/ui/components/user-list/user-list-item/component.jsx

```diff
@@ -335,11 +410,15 @@ class UserListItem extends Component {
         </span>
         <span className={styles.userNameSub}>
           {userNameSub}
-          {(user.isLocked && (disablePrivateChat || disableCam || disableMic || lockedLayout || disablePublicChat)) ?
-            <span> {(user.isCurrent? " | " : null)}
+          {(user.isLocked && (disablePrivateChat
+            || disableCam
+            || disableMic
+            || lockedLayout
+            || disablePublicChat)) ?
+            <span> {(user.isCurrent ? ' | ' : null)}
               <Icon iconName='lock' />
               {intl.formatMessage(messages.locked)}
```

Comment:
Can't we have this as a description of the icon instead of this ugly text when the user is locked? The icon alone looks way better/clean.